articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md

```diff
@@ -0,0 +1,485 @@
+
+    ![Subnets button in virtual networks blade](images/azs-public-browser-virtual-network-subnets-button.png)
+
+3. On the **Subnets** page, click the **Gateway Subnet** button.
+
+    ![Gateway subnet button in virtual networks blade](images/azs-public-browser-virtual-network-gateway-subnet-button.png)
+
+4. Fill in the **Address range** and add a route table if required, then click **OK**. The subnet should now have appeared.
+
+    ![Add gateway subnet blade](images/azs-public-browser-virtual-network-add-gateway-subnet.png)
+
+## Create the VPN gateway in Public Azure
+
+1. In the favourites panel, select **Create a resource**.
+
+   ![Create a resource option in favourites panel](images/azs-public-browser-create-resource-button.png)
+
+2. In the **New** blade, select **Networking**.
+
+   ![Networking option in New blade](images/azs-public-browser-new-networking-button.png)
+
+3. In the **Featured** section, select **See all**, then **Virtual network gateway**.
+
+   ![Create new networking resource](images/azs-public-browser-new-virtual-network-gateway.png)
+
+4. Click **Create**.
+
+5. In the **Create virtual network gateway** blade, enter the following information:
+
+   - **Subscription** - This is your Microsoft Azure subscription.
+
+   - **Name** - The name of the virtual network gateway.
+
+   - **Region** - The region to place the virtual network gateway in.
+
+   - **Gateway type** - The type of virtual network gateway to create.
+
+       - To connect with an existing ExpressRoute circuit, select "ExpressRoute".
+
+   - **VPN type** - The type of VPN you can choose depends on the make and model of your VPN device, and the kind of VPN connection you intend to create. Choose a route-based gateway if you intend to use point-to-site, inter-virtual network, or multiple site-to-site connections; if you are creating a VPN type gateway to coexist with an ExpressRoute gateway; or if you need to use IKEv2. Policy-based gateways support only IKEv1.
+
+   - **SKU** - Route-based VPN gateway types are offered in three SKUs: Basic, Standard, and High performance. You must select Standard or High performance if you are creating the network to coexist with an ExpressRoute gateway. You must select High performance SKU to enable active-active mode. You can find more information about SKUs here: [Azure Stack SKUs](https://docs.microsoft.com/en-gb/azure/vpn-gateway/vpn-gateway-about-skus-legacy).
+
+   - **Virtual Network** - This is the virtual network that you created earlier in Public Azure.
+
+   - **Public IP address** - The public IP address to assign to this virtual network gateway. Only dynamic public IP addresses are supported.
+
+       - Click **Create new**.
+
+       - In the **Public IP address name** field, enter a public IP address name.
+
+   - **Enable active-active mode** - The gateway SKU must be either **VpnGw1**, **VpnGw2**, **VpnGw3**, **VpnGw1AZ**, **VpnGw2AZ**, **VpnGw3Az** or HighPerformance (legacy SKU).
```

Comment:
If you want to enable active-active mode, the gateway SKU must be one of: **VpnGw1**, **VpnGw2**, **VpnGw3**, **VpnGw1AZ**, **VpnGw2AZ**, **VpnGw3Az** or **HighPerformance** (legacy SKU)

(only use bold if these values show up somewhere in the UI - if it's just for information, no bold is needed)